scripts/tests3.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  CreateBucketCommand,
+  GetObjectCommand,
+  GetObjectCommandInput,
+  HeadObjectCommand,
+  HeadObjectCommandInput,
+  PutObjectCommand,
+  PutObjectCommandInput,
+  DeleteObjectCommand,
+  DeleteObjectCommandInput,
+  DeleteBucketCommand,
+  DeleteBucketCommandInput,
+} from '@aws-sdk/client-s3'
+import { getSignedUrl as awsGetSignedUrl } from '@aws-sdk/s3-request-presigner'
+import { getPrismaClient } from 'api/src/lib/db'
+import AWS from 'api/src/lib/aws'
+
+const REPORTING_DATA_BUCKET_NAME = `${process.env.REPORTING_DATA_BUCKET_NAME}`
+
+export default async () => {
+  const s3 = AWS.getS3Client()
+  await getPrismaClient()
+  const key = "12345"
+
+  console.log("Create bucket")
+  try {
+    await s3.send(new CreateBucketCommand({
+      Bucket: REPORTING_DATA_BUCKET_NAME,
+    }))
+    console.log("-- Bucket created")
+  } catch (error) {
+    if (error.Code == "BucketAlreadyOwnedByYou") {
+      console.log("-- Bucket already exists")
+    } else {
+      console.log(error)
+    }
+  }
+
+  const baseParams: GetObjectCommandInput = {
+    Bucket: REPORTING_DATA_BUCKET_NAME,
+    Key: key,
+  }
+  try {
+    console.log("Create signed URL")
+    const url = await awsGetSignedUrl(s3, new GetObjectCommand(baseParams), {
+      expiresIn: 60,
+    })
+    console.log("-- signed url", url)
+
+    console.log("Upload file")
+    const uploadParams: PutObjectCommandInput = {
+      Body: "tests3.ts",  // Upload this file as a test
+      Bucket: REPORTING_DATA_BUCKET_NAME,
+      Key: key,
+    }
+    await s3.send(new PutObjectCommand(uploadParams))
+
+    console.log("-- Look at uploaded file")
+    const headParams: HeadObjectCommandInput = {
+      Bucket: REPORTING_DATA_BUCKET_NAME,
+      Key: key,
+    }
+    const response = await s3.send(new HeadObjectCommand(headParams))
+    console.log("--", response)
+
+    console.log("Delete file")
+    const deleteParams: DeleteObjectCommandInput = {
+      Bucket: REPORTING_DATA_BUCKET_NAME,
+      Key: key,
+    }
+    await s3.send(new DeleteObjectCommand(headParams))
```

Comment:
Just realizing after my last comment - since this uses a prod bucket, maybe omit the delete after all. Or use a different bucket just for the test script. Sorry, bad idea on my part!

Reply:
I added a `test` prefix and generated uuid for a suffix to make sure we don't mess with any real data.